frontend/src/app/_services/schedule.service.ts

```diff
@@ -22,20 +22,33 @@ export class ScheduleService {
   get_data() : Observable<ScheduleData>{
 
     // real is currently broken, we'll start with fake
-    return of<ScheduleData>({program, people});
+    return of<ScheduleData>({program: (program as ProgramItem[]), people: (people as ProgramPerson[])});
     // this is real
     // return this.http.get<ScheduleData>(`${environment.backend}/schedule`);
   }
 
   get_schedule(filters: ProgramFilter = {}): Observable<[string, string, ProgramItem[]]> {
-    const munged_filters: { tags?: (_:string[]) => boolean, loc?: (_:string[]) => boolean, date?: (_:string) => boolean } = {}
-    Object.keys(filters).forEach(f => {
-        munged_filters[f] = pf => filters[f].some(f => pf.includes(f) || pf === f)
-    });
+    const munged_filters: ((programItem: ProgramItem) => boolean)[] = [];
+    if (filters.tags && filters.tags.length > 0) {
```

Comment:
don't love that we've specified these here explicitly - there's ways to do typing with keyof or whatever the keyword is